src/testRunner/unittests/tsserver/projectErrors.ts

```diff
@@ -583,8 +583,7 @@ describe("unittests:: tsserver:: projectErrors:: dont include overwrite emit err
         const session = new TestSession(host);
         openFilesForSession([f1], session);
 
-        const projectService = session.getProjectService();
-        const projectFileName = projectService.inferredProjects[0].getProjectName();
+        const projectFileName = session.getProjectService().inferredProjects[0].getProjectName();

```

Comment:
So is `projectService` not safe to reuse between requests now?

Reply:
It is but I changed this so it’s easier to track any other usages